docs/advanced_topics/accessibility_considerations.md

```diff
@@ -220,7 +220,7 @@ Occasional Wagtail users may not be aware of your site’s content guidelines, o
 
 ### Readability
 
-Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/vixdigital/wagtail-readinglevel) as a score displayed in rich text fields.
+Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/neon-jungle/wagtail-readability) as a score displayed in rich text fields.
```

Comment:
wagtail-readability is a different package that uses Hallo.js. Let's point to the Torchbox fork of wagtail-readinglevel which is more up-to-date.

```suggestion
Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/torchbox-forks/wagtail-readinglevel) as a score displayed in rich text fields.
```

Reply:
updated the link